configs/example/gem5_library/checkpoints/simpoints-se-restore.py

```diff
@@ -0,0 +1,70 @@
+from gem5.simulate.exit_event import ExitEvent
+from gem5.simulate.simulator import Simulator
+from gem5.utils.requires import requires
+from gem5.components.cachehierarchies.classic.\
+    private_l1_private_l2_cache_hierarchy import(
+    PrivateL1PrivateL2CacheHierarchy
+)
+from gem5.components.boards.simple_board import SimpleBoard
+from gem5.components.memory import DualChannelDDR4_2400
+from gem5.components.processors.simple_processor import SimpleProcessor
+from gem5.components.processors.cpu_types import CPUTypes
+from gem5.isas import ISA
+from gem5.resources.resource import Resource
+
+from gem5.components.processors.simpoint import SimPoint
+from pathlib import Path
+
+cache_hierarchy = PrivateL1PrivateL2CacheHierarchy(
```

Comment:
*Here* is a good place to use the more complex hierarchy. This is great. Make a note in a comment that you can change the hierarchy :)